config/environments/development.rb

```diff
@@ -45,6 +45,23 @@
     Bullet.rails_logger = true
     Bullet.add_footer = true
   end
+
+  # Setup mailer
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default charset: "utf-8"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.